lisa/sut_orchestrator/util/device_pool.py

```diff
@@ -0,0 +1,63 @@
+from typing import Any, List, Optional
+
+from lisa.sut_orchestrator.util.schema import HostDevicePoolSchema, HostDevicePoolType
+from lisa.util import LisaException, SkippedException
+
+
+class BaseDevicePoolImpl:
```

Comment:
What's the `Impl` for? Call it like `BaseDevicePool`.

Reply:
Sure Chi, Will change the class name. I used Impl as shortform of implementation.

Reply:
@squirrelsc Please check the latest push.